datasource/src/main/scala/astraea/spark/rasterframes/datasource/rastersource/RasterSourceDataSource.scala

```diff
@@ -40,6 +43,11 @@ object RasterSourceDataSource {
   final val PATHS_PARAM = "paths"
   final val BAND_INDEXES_PARAM = "bandIndexes"
   final val TILE_DIMS_PARAM = "tileDimensions"
+  final val PATH_TABLE_PARAM = "pathTable"
+  final val PATH_TABLE_COL_PARAM = "pathTableColumn"
+
+  /** Container for specifying where to select raster paths from. */
+  case class PathColumn(tableName: String, columnName: String)
```

Comment:
I think the user may want to specify several columns from the given table. E.g.  red_uri, nir_uir, swir_uri ... 

Where the use case would be many single-band images that are aligned with same spatial metadata. e.g. PDS buckets.

Reply:
Aaaaahh.... good call. Will think on that.

Reply:
Here's the API and resulting schema, so far:

`pathsTable`:
```
+--------------------+--------------------+--------------------+
|                  B1|                  B2|                  B3|
+--------------------+--------------------+--------------------+
|file:/Users/sfitc...|file:/Users/sfitc...|file:/Users/sfitc...|
+--------------------+--------------------+--------------------+
```

```scala
val df = spark.read.rastersource
  .fromTable("pathsTable", "B1", "B2", "B3")
  .withTileDimensions(128, 128)
  .load()
```

```
root
 |-- B1_path: string (nullable = true)
 |-- B2_path: string (nullable = true)
 |-- B3_path: string (nullable = true)
 |-- B1: struct (nullable = true)
 |    |-- tileContext: struct (nullable = false)
 |    |    |-- extent: struct (nullable = false)
 |    |    |    |-- xmin: double (nullable = false)
 |    |    |    |-- ymin: double (nullable = false)
 |    |    |    |-- xmax: double (nullable = false)
 |    |    |    |-- ymax: double (nullable = false)
 |    |    |-- crs: struct (nullable = false)
 |    |    |    |-- crsProj4: string (nullable = false)
 |    |-- tile: tile (nullable = false)
 |-- B2: struct (nullable = true)
 |    |-- tileContext: struct (nullable = false)
 |    |    |-- extent: struct (nullable = false)
 |    |    |    |-- xmin: double (nullable = false)
 |    |    |    |-- ymin: double (nullable = false)
 |    |    |    |-- xmax: double (nullable = false)
 |    |    |    |-- ymax: double (nullable = false)
 |    |    |-- crs: struct (nullable = false)
 |    |    |    |-- crsProj4: string (nullable = false)
 |    |-- tile: tile (nullable = false)
 |-- B3: struct (nullable = true)
 |    |-- tileContext: struct (nullable = false)
 |    |    |-- extent: struct (nullable = false)
 |    |    |    |-- xmin: double (nullable = false)
 |    |    |    |-- ymin: double (nullable = false)
 |    |    |    |-- xmax: double (nullable = false)
 |    |    |    |-- ymax: double (nullable = false)
 |    |    |-- crs: struct (nullable = false)
 |    |    |    |-- crsProj4: string (nullable = false)
 |    |-- tile: tile (nullable = false)
```